renderer/brave_content_renderer_client.cc

```diff
@@ -75,6 +77,11 @@ void BraveContentRendererClient::RenderFrameCreated(
     new brave_search::BraveSearchRenderFrameObserver(
         render_frame, content::ISOLATED_WORLD_ID_GLOBAL);
   }
+
+  if (brave_ads::features::IsRequestAdsEnabledApiEnabled()) {
+    new brave_ads::BraveAdsRenderFrameObserver(
```

Comment:
this needs ads buildflag in source and gn

Reply:
Fixed, buildflag was removed here: https://github.com/brave/brave-core/pull/9769